server2/Arcadia.Assistant/Arcadia.Assistant.Sharepoint/SharepointSynchronizator.cs

```diff
@@ -0,0 +1,171 @@
+﻿namespace Arcadia.Assistant.Sharepoint
+{
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Threading;
+    using System.Threading.Tasks;
+
+    using CalendarEvent;
+
+    using Employees.Contracts;
+
+    using ExternalStorages.Abstractions;
+
+    using Microsoft.Extensions.Logging;
+
+    using SickLeaves.Contracts;
+
+    using Vacations.Contracts;
+
+    using WorkHoursCredit.Contracts;
+
+    public sealed class SharepointSynchronizator
+    {
+        #region ctor
```

Comment:
Especially this.

Reply:
Yes. Especially this. You can close it and save screen size for necessary code.

Reply:
Or you can just scroll several lines to the bottom and also save screen size. But without any non-functional noize in the code.

Reply:
Or you can just clsoe the region and be happy without current problem out code noize.